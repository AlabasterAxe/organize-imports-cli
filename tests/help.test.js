const runCli = require("./run-cli");

describe("--help", () => {
  runCli(".", ["--help"]).test();
});
