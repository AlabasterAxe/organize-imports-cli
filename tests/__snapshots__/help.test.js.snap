// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`--help (status) 1`] = `0`;

exports[`--help (stderr) 1`] = `""`;

exports[`--help (stdout) 1`] = `
"
Usage: organize-imports-cli files...

Files can be specific ts and js files or tsconfig.json, in which case the whole project is processed.

Files containing the substring \\"// organize-imports-ignore\\" are skipped.

"
`;

exports[`--help (write) 1`] = `Array []`;
